comictalker/__init__.py

```diff
@@ -30,7 +32,20 @@ def get_talkers(version: str, cache: pathlib.Path) -> dict[str, ComicTalker]:
             if obj.id != talker.name:
                 logger.error("Talker ID must be the same as the entry point name")
                 continue
-            talkers[talker.name] = obj
+            try:
+                if parse(version) >= parse(obj.comictagger_min_ver):
```

Comment:
Move this `parse(version)` out of the loop, we don't need to parse this again for every plugin